integration_test/workflow_tests/test_packages/ingest/test_no_granules.py

```diff
@@ -0,0 +1,95 @@
+import json
+import time
+import uuid
+from unittest import TestCase
+
+import boto3
+
+import helpers
+from requests import Session
+
+
+class TestNoGranules(TestCase):
+    def test_no_granules_passes(self):
+        """
+        If no granules are provided, should not store anything in DB.
+        """
+        # Set up Orca API resources
+        # ---
+        my_session = Session()
+        my_session.mount(
+            helpers.api_url,
+            helpers.DNSResolverHTTPSAdapter(
+                helpers.aws_api_name, helpers.API_LOCAL_HOST
+            ),
+        )
+
+        provider_id = uuid.uuid4().__str__()
+        provider_name = uuid.uuid4().__str__()
+
+        collection_name = uuid.uuid4().__str__()
+        collection_version = uuid.uuid4().__str__()
```

Comment:
Not really a valid version persey. The version is usually a 0 padded number converted to a string like `007` or `061` for example.

Reply:
That's a Cumulus requirement, isn't it? Don't feel like we should couple to those.

Reply:
More along the lines of the type of layout most DAACs use for collections but, but considering we get that data from Cumulus may not necessarily be a bad thing. More of an observation than anything. No real need to change the random sting you are generating now.